### DISCUSSION TITLE
Were can I find custom SMTP configuration?

### DISCUSSION THREAD

**Discussion by devops-versai** [supabase/supabase#8808] [Questions] [answered]
We are using supabase. But after the recent release in supabase, I am not able to find smtp settings in the authentication dashboard. 
<img width="1440" alt="Screenshot 2022-09-06 at 4 40 32 PM" src="https://user-images.githubusercontent.com/99470670/188621762-d9c6978c-858e-42ff-8754-32d2368dac5f.png">

**Comment by GaryAustin1** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/54564956/188682739-b4d304e6-10dd-4746-98c8-456230892bc4.png)

**Comment by JulianSwales** [comment]
Soo, where is this hiding now? I don't have that option in my Email Templates.. 
![image](https://user-images.githubusercontent.com/66487981/235999255-75d245cb-106f-49fb-8abd-4a6c31e8c3e5.png)

**Reply by aokigit** [reply]
Project Settings > Auth > SMTP Settings

<img width="1920" alt="3fJ93" src="https://user-images.githubusercontent.com/20214774/236273618-b4ec28d8-d767-44f2-b818-0ac85612fc81.png">
